.github/workflows/ci.yml

```diff
@@ -0,0 +1,81 @@
+name: Continuous Integration
+
+on: [push, pull_request]
```

Comment:
I would spec here build only the master and tags on push, all other on pull request

Reply:
Resolved in another way to prevent duplicate builds. I still want to run on push for feature branches so the ci is triggered.